logical/src/main/java/org/apache/drill/common/logical/StoragePluginConfig.java

```diff
@@ -29,6 +31,13 @@
   // DO NOT include enabled status in equality and hash
   // comparisons; doing so will break the plugin registry.
   private Boolean enabled;
+  protected final Integer reconnectRetries;
```

Comment:
This property is used for the Splunk plugin only, maybe it makes sense to move it there instead of declaring in a base class?

Reply:
But some plugins might not use this property, or it may be invalid for them, for example, there is no reason to use it for `sys` or `cp` plugins, but they all will have it...